### ISSUE TITLE
Example of specifying a custom launch template for a node-group

### ISSUE THREAD

**Issue by dimitry12** [weaveworks/eksctl#1313]
In https://github.com/weaveworks/eksctl/issues/479#issuecomment-488293565 @errordeveloper said that custom launch template support has been merged.

I've looked through https://github.com/weaveworks/eksctl/pull/743 but I failed to figure out how to specify a custom launch template when creating a new node-group.

I am in search of a YAML example. Thank you!

For completeness: my use-case is launching a nodegroup with each node having a second EBS volume attached, with second EBS volume being created from a snapshot. I am confident that I know what I'm doing, even if it seems non-k8s-like :D

**Comment by tomhuang12** [comment]
Any updates on this?

**Comment by michaelbeaumont** [comment]
To be clear, there is no support right now for specifying custom launch templates.

**Comment by cPu1** [comment]
Support for custom launch templates in managed nodegroups is in progress, it will be out in the next few days.
https://github.com/weaveworks/eksctl/pull/2544

**Comment by tomhuang12** [comment]
@cPu1 That's awesome! In your PR, you mentioned that custom AMI can be used for managed node group. Does that mean this feature: https://github.com/aws/containers-roadmap/issues/741?

**Comment by cPu1** [comment]
> @cPu1 That's awesome! In your PR, you mentioned that custom AMI can be used for managed node group. Does that mean this feature: [aws/containers-roadmap#741](https://github.com/aws/containers-roadmap/issues/741)?

Yes, that's correct. More details about this feature will be out on https://eksctl.io shortly.
